### PR TITLE
bpo-43215: Document Happy Eyeballs args of asyncio.open_connection

### PR DESCRIPTION
**Pull request by illia-v**
https://bugs.python.org/issue43215

### CODE REVIEW

File: Doc/library/asyncio-stream.rst [hunk 1]
```diff
@@ -66,6 +67,10 @@ and work with streams:
    The rest of the arguments are passed directly to
    :meth:`loop.create_connection`.
 
+   .. versionadded:: 3.8
```
Comment (serhiy-storchaka) [inline Doc/library/asyncio-stream.rst]: Usually `versionadded` directives are ordered from older to newer versions. In this case it should be below `versionadded:: 3.7`.

Comment (illia-v) [reply Doc/library/asyncio-stream.rst]: They are the other way around in [`loop.create_connection` docs](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_connection). Should I reorder them here?

Comment (serhiy-storchaka) [reply Doc/library/asyncio-stream.rst]: You can include that change in this PR. But if there are other mis-ordered directives it is better to create a separate PR.

Comment (illia-v) [reply Doc/library/asyncio-stream.rst]: I ordered the directives, thanks

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**